spec/features/case_log_spec.rb

```diff
@@ -218,4 +218,26 @@ def fill_in_number_question(case_log_id, question, value)
       end
     end
   end
+
+  describe "Conditional questions" do
+    context "given a page where some questions are only conditionally shown, depending on how you answer the first question" do
+      it "initially hides conditional questions" do
+        visit("/case_logs/#{id}/armed_forces")
+        expect(page).not_to have_selector("#armed_forces_injured_div")
+      end
+
+      it "shows conditional questions if the required answer is selected and hides it again when a different answer option is selected", js: true do
```

Comment:
I feel like this , one that tests the conditional questions appear given selection of the appropriate option and another that tests that the conditional questions are hidden when you select a different option given that your current selection is revealing the conditional questions. Currently it feels like this tests is doing two sets of acts and asserts or is that fine since this is a feature test?

Reply:
It is doing multiple things yes. My reasoning for making it a "single test" was mostly that this is sort of a single feature/user journey, so you can't really test the second step here without doing the first. Duplicating the first just to have it be it's own test didn't really seem worth it?

Reply:
That's fair enough, I think if it was a unit test I'd be more concerned about it being a single test 🙂